src/service.rs

```diff
@@ -0,0 +1,10 @@
+use tarpc;
+
+/// This is the service definition. It looks a lot like a trait definition.
+#[tarpc::service]
```

Comment:
we should undoubtedly autogenerate this table. Could you add a TODO and an issue?